pandas/tests/indexes/test_multi.py

```diff
@@ -1425,8 +1448,6 @@ def test_equals_multi(self):
 
         self.assertFalse(self.index.equals(self.index[:-1]))
 
-        self.assertTrue(self.index.equals(self.index._tuple_index))
```

Comment:
See the discussion here: https://github.com/pandas-dev/pandas/pull/13986#r74693208. We changed `Index.equals` behaviour to return False for everything that is not an Index. I was not in favor of that change, but since we only changed that in 0.19.0, we should keep to that decision I think